app/modules/configuration/resources.py

```diff
@@ -35,22 +33,17 @@ class EDMConfigurationDefinition(Resource):
     """
 
     def get(self, target, path):
-        response = current_app.edm.request_passthrough(
-            'configurationDefinition.data', 'get', {}, path, target
+        current_app.edm.pseudo_initialize(target)
+        data = current_app.edm._get(
```

Comment:
should not call private _get from outside edm. 
Surely this would be better structured by adding ensure_initialised, decode_as_object and decode_as dict as default params to request_passthrough.
In request_passthrough, if ensure_initialized is true, call _ensure_initialized, else call _pseudo_initialize